spec/Appendix B -- Grammar Summary.md

```diff
@@ -94,10 +94,14 @@ lines and uniform indentation with {BlockStringValue()}.
 Document : Definition+
 
 Definition :
+  - DocumentDirective
```

Comment:
```suggestion
  - DocumentDirectives
```

Reply:
No, `DocumentDirective` (singular) is really what should be stated in grammar.

Reply:
I see now
Then I think it makes more sense to do
Document: Directives[Const] Definition+

Reply:
This will prevent you from inserting document level directives into middle of document.
This may be problematic if you *concatenate* more documents into one.

Reply:
Moreover there is chicken egg problem,  when you wish define document directive you cannot use it in same document

Reply:
Oh, you are right…